armada/handlers/k8s.py

```diff
@@ -92,3 +94,30 @@ def delete_namespace_pod(self, name, namespace="default", body=None):
 
         return self.client \
             .delete_namespaced_pod(name, namespace, body)
+
+    def wait_for_pod_redeployment(self, old_pod_name, namespace):
+        base_pod_pattern = re.compile('^(.+)-[a-zA-Z0-9]+$')
+        if not base_pod_pattern.match(old_pod_name):
+            LOG.error(
+                'Could not identify new pod after purging %s', old_pod_name)
+            return
+        pod_base_name = base_pod_pattern.match(old_pod_name).group(1)
+
+        new_pod_name = ''
+        w = watch.Watch()
+        for event in w.stream(self.client.list_namespaced_pod, namespace):
```

Comment:
@gardlt only thing I wanted to ask you about was giving some timeout here. i'm just worried if for some reason a job/service/whatever to redeploy the pod isn't running, the stream will sit open forever. the stream can take a timeout but I wasn't sure what value to use. maybe pass the timeout from the chart? and set some other default?